CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.5.6] - 2018-10-23
+### Fixed
+- Cambia el endpoint de WebSocket (utilizado para actualizar el estado de la transacción)
```

Comment:
This should be worded from the POV of the integrator and not from our own POV. I suggest something like “Corrige sincronización (websocket) con el avance del pago en la app Onepay”